validation/telemetry/update.4.ready.json

```diff
@@ -18,7 +18,8 @@
     "reason" : "ready",
     "targetChannel" : "default",
     "targetVersion" : "56.0a1",
-    "targetBuildId" : "20080811053724"
+    "targetBuildId" : "20080811053724",
+    "targetDisplayVersion" : "56.0 Beta 1"
```

Comment:
I think what we are aiming at is `56.0b1` rather than `56.0 Beta 1`

@sylvestre can you confirm that it is the format of displayVersion?

Reply:
As mentioned [in the bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1405614#c5), that's the information reported by the update MAR. Telemetry just takes it and reports it back.

If you want/need that to change, you should reach out to the Update team and file a bug to change it.